bin/coverage

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 set -eu
 
-# Send code coverage report to coveralls.io
+# Merge all code coverage files into one
 gover ./code-coverage ./code-coverage/gover.coverprofile
-
-goveralls -coverprofile code-coverage/gover.coverprofile -service=travis-ci -repotoken "$COVERALLS_TOKEN"
+sed -i 's/\([0-9]\)\(code.cloudfoundry.org\)/\1\n\2/g' ./code-coverage/gover.coverprofile
```

Comment:
I've noticed the same issue while working on #92 , it looks that happens when it's being specified the coverfile to a single file : https://github.com/cloudfoundry-incubator/quarks-job/blob/master/bin/test-unit#L9 this happens. 

To note this is the workaround used in quarks-utils: https://github.com/cloudfoundry-incubator/quarks-utils/blob/master/bin/coverage#L5